### DISCUSSION TITLE
Can I play test a map from a specific time in the editor?

### DISCUSSION THREAD

**Discussion by LOKSTED** [ppy/osu#13679] [Q&A] [answered]
Like the one on the Stable Editor? Looked everywhere but I can't find :/

**Comment by smoogipoo** [comment, marked as answer]
Not yet. Tracked in https://github.com/ppy/osu/issues/7008

**Reply by LOKSTED** [reply]
oh that poor guy had been waiting since 2019. 
I know you have a lot of suggestions already but what do you think about some kind of a mod that allows you to play a specific part of a song? I've been using the test button on stable to do just that but it would be cool to be able to play with mods too since the editor doesn't have mod testing.

**Reply by frenzibyte** [reply]
Also tracked, in here https://github.com/ppy/osu/discussions/13013
